### PR TITLE
Fix regressions introduced in previous releases

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
#283 changed the role of the menu items in the location field dropdown. This prevented the onFocus handler from ensuring their click handler fires before the dropdown disappears, resulting in menu items not being clickable.

This PR also updates the geocoder to work in more build environments, updating to use the latest lonlat.

Also includes a few related patches for location-field to make integration into otp-rr easier.

### CODE REVIEW

File: packages/location-field/src/styled.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ export const DropdownButton = styled(BaseButton)`
 `;
 
 export const DropdownContainer = styled.span`
-  position: relative;
+  position: contents;
```
Comment (binh-dam-ibigroup) [inline packages/location-field/src/styled.js:25]: One of these CSS changes causes the dropdown suggestion list to be limited by the width of the text field, and I don't think we want that. On desktop views, the dropdown width should extend to accommodate as much of its contents as possible.
image

File: packages/location-field/src/index.js [hunk 2]
```diff
@@ -430,6 +430,9 @@ class LocationField extends Component {
         .filter(feature => feature.properties.source !== "transit")
         .slice(0, suggestionCount);
 
+      const nonOtherFeaturesPresent =
```
Comment (binh-dam-ibigroup) [inline packages/location-field/src/index.js:433]: Rename to `standardFeatures`?

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Please advise regarding changes to the width of the dropdown menu in desktop view.

File: packages/location-field/src/index.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/location-field/src/index.js]: This is much better thank you! Renamed in 026fb0c

File: packages/location-field/src/styled.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/location-field/src/styled.js:25]: I had to make some changes in https://github.com/opentripplanner/otp-react-redux/pull/462 to accommodate this but thank you for suggesting in the end it is much nicer to be able to display the entire title! Updated in be62bb29b9db033a82706971d3c4031a337aef23

File: packages/location-field/src/index.js [hunk 3]
```diff
@@ -430,7 +430,9 @@ class LocationField extends Component {
         .filter(feature => feature.properties.source !== "transit")
         .slice(0, suggestionCount);
 
-      const nonOtherFeaturesPresent =
+      // If no categories of features are returned, this variable is used to
+      // avoid displaying headers
+      const nonStandardFeaturesPresent =
```
Comment (binh-dam-ibigroup) [inline packages/location-field/src/index.js:435]: I thought stops and stations fall under "standard" features?

Comment (binh-dam-ibigroup) [reply packages/location-field/src/index.js:435]: Rename to `standardFeatures`, I think.

**Review by binh-dam-ibigroup — APPROVED** [review]
Good to go after renaming variable for standard features.

File: packages/location-field/src/index.js (hunk 3 above)
Comment (miles-grant-ibigroup) [reply packages/location-field/src/index.js]: I don't think we have any other features, I read it as "other" is the standard feature and the stations and stops were special. I've renamed to `transitFeatures` in 8ea983c6ab004d320de2da88e76bc66ee2be95e3 which I think is more clear.

**Review by philip-cline — APPROVED** [review]

**Pull request merged**